### ISSUE TITLE
`wordSplitRegex` should be configurable via Text Entity Recognition and Text Entity Relations configuration screen

### ISSUE THREAD

**Issue by seveibar** [UniversalDataTool/universal-data-tool#374]
i.e. somebody should be able to create configuration like the one shown below from `Setup > Configure`

![image](https://user-images.githubusercontent.com/1910070/98976334-37f75980-24e5-11eb-81a9-24fe4826cb41.png)

**Comment by Ownmarc** [comment]
I didn't check the code you did to implement this, but we should be able to override the whole regex or simply add to it. This way, one could set its own regex to split words while someone else could simply add a few characters to the default regex.

**Comment by seveibar** [comment]
Agreed. IMO I think we should do that at the interface layer rather than the data/format error (we could actually have a cool kind of mini regex test interface against the data idk)
